eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -40,5 +40,10 @@
     <!-- Transitive dependency of Roslyn dependencies. -->
     <UsagePattern IdentityGlob="System.Text.Json/8.0.0*" />
     <UsagePattern IdentityGlob="System.Text.Encodings.Web/8.0.0*" />
+
+    <!-- Dependencies that are used by ASP.NET Core source build CI but not used in full VMR build. -->
+    <UsagePattern IdentityGlob="System.Composition/1.0.*" />
```

Comment:
Instead of adding a new section...

System.Composition and IO.Pipelines are coming from the analyzers and better fit in https://github.com/dotnet/aspnetcore/pull/56771/commits/c6e6f2f868799e5b7e48afe7da1f4eb5ea1b589f#diff-f88ce4026366ce74fb1a981785c542757617abf4393a33ca34465a5e9a0dd343L21.

System.Security.Cryptography.Pkcs is a transitive dependency from the 5.0 aspnetnetcoreappref.  I think this could go in https://github.com/dotnet/aspnetcore/pull/56771/commits/c6e6f2f868799e5b7e48afe7da1f4eb5ea1b589f#diff-f88ce4026366ce74fb1a981785c542757617abf4393a33ca34465a5e9a0dd343R13